README.md

```diff
@@ -20,7 +24,7 @@ Tutorials are organised by data access method, rather than data type. Within the
 | [Tilde](./Tilde) | Shows how to retrieve data from GeoNet's [Tilde API](https://tilde.geonet.org.nz/v3/api-docs/). These turorails apply to **DART**, **envirosensor**, and **tsunami gauge (down-sampled)** data. Tutorials cover Tilde's data, stats, and data summary APIs.
 
 ## How to Run Tutorials
-The file [**environment.yml**](environment.yml) ensures that you have the correct Python environment to run the data tutorials. It allows you to install the correct Python packages, and where appropriate package versions. We use this environment when writing and reviewing notebooks. If you use a different environment, tutorials may work, but we cannot guarantee it.
+The file [**environment.yml**](./environment.yml) ensures that you have the correct Python environment to run the data tutorials. It allows you to install the correct Python packages, and where appropriate package versions. We use this environment when writing and reviewing notebooks. If you use a different environment, tutorials may work, but we cannot guarantee it.
```

Comment:
this is not in a folder, can you please change the link back to the original?
` [**environment.yml**](environment.yml)`